src/applications/survivor-dependent-education-benefit/22-5490/config/form.js

```diff
@@ -205,6 +205,10 @@ const formConfig = {
                           'Please enter a valid entry. Acceptable entries are letters, spaces and apostrophes.',
                         );
                       }
+                    } else if (!isValidName(field)) {
```

Comment:
These `else if` statements redundant. Seems like just an `else` would work for each of these. Up to you, I'll approve the PR still.

Reply:
Made a change to remove the else statement as it was causing an error on a valid entry so the else if explicitly catches an invalid name